robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -1064,75 +1064,6 @@ def _resolve_interpreter(self, params=None) -> ActionResultDict:
         # i.e.: no error but we couldn't find an interpreter.
         return {"success": True, "message": "", "result": None}
 
-    @command_dispatcher(
```

Comment:
I think that we no longer use the `robocorp_code.locators_db` module either, so, that module should probably be removed (as well as its test).

And the `_LocatorsInThreadAPI` class in this module seems to be unused too...